distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+==== Directory Permissions
```

Comment:
It is _not_ a prerequisite. You actually can't change the permissions on `/etc` and `/data` until the zip has been extracted (section 2.2).  Which is why I would recommend moving this to a later section that addresses hardening/security.